src/config/main.lua.d/40-device-defaults.lua

```diff
@@ -5,6 +5,7 @@ device_defaults.properties = {
   -- when set to false, default nodes and routes are selected based on
   -- their priorities and any runtime changes do not persist after restart
   ["use-persistent-storage"] = true,
+  ["default-volume"] = 0.03,
```

Comment:
Does this set all devices, both input and output, to 3%?

Reply:
Not 3%, the values are not 0->1, it's about 40%. But yes, there is no option do set just input default volume, I could make a upstream PR to change that, but I figure most people have buttons on their keyboard to change output volume easily.

Reply:
That sounds good to me! Default output volume at 40% should also be reasonable.

Reply:
Looks like it drops all default volume values to 30%

Reply:
@XV-02 ready to approve?

Reply:
He is testing it with the install from an iso, just to be sure that this will work for users who purchase our machines.

Reply:
Looks like all volume levels (input and output) default to ~30%.